test/functions/math.test.cpp

```diff
@@ -173,3 +173,15 @@ TEST_CASE("pow with string base", "[functions][math][pow][string]") {
             "08862990907940498760592041482791966715057053380053636634477979263"
             "4601932346249911644260403539535769743430516736");
 }
+
+TEST_CASE("Sqrt of big integers", "[functions][math][sqrt][big]") {
+    BigInt big_num1 = "3479680743635798438602889954485038919559460388450";
```

Comment:
I think this is where you're getting compilation errors. Please see #3 for workarounds.